tests/lit-tests/func_template_12.ispc

```diff
@@ -0,0 +1,21 @@
+// RUN: %{ispc}  %s --emit-llvm-text --target=host --nostdlib -o - | FileCheck %s
```

Comment:
Would be good to spell it out what you are trying to test here.

Reply:
Added.